packages/ovh-manager-core/ovh-manager-core.js

```diff
@@ -12,22 +19,25 @@ export default angular
     'ng-at-internet',
   ])
   .constant('constants', {})
+  .constant('LANGUAGES', LANGUAGES)
+  .constant('MANAGER_URLS', MANAGER_URLS)
+  .constant('REDIRECT_URLS', REDIRECT_URLS)
+  .constant('TARGET', 'EU')
```

Comment:
We shouldn't manage it here.

I think we have to deliver a SPA without any reference to the target, and the difference between targets should be managed by API :)